components/log-viewer-webui/server/package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "log-viewer-server",
+  "name": "log-viewer-webui-server",
   "version": "0.1.0",
   "description": "",
   "main": "src/main.js",
   "scripts": {
-    "dev": "NODE_ENV=development nodemon src/main.js",
-    "lint:check": "npx eslint --no-eslintrc --config package.json src",
-    "lint:fix": "npx eslint --fix --no-eslintrc --config package.json src",
     "start": "NODE_ENV=production node src/main.js",
```

Comment:
Would it be better to rename this script as `prod` and the one below for development as `start`? That way we will be consistently using `start` as the name for all development scripts across all repos.